sopt/src/main/scala/dagr/sopt/cmdline/ClpReflectiveBuilder.scala

```diff
@@ -252,8 +252,10 @@ private[sopt] class ClpArgument(declaringClass: Class[_],
     */
   private def prettyNameValue(value: Any): String = {
     Option(value) match {
-      case Some(v) if isSensitive => String.format("--%s ***********", longName)
-      case Some(v) => String.format("--%s %s", longName, v.toString)
+      case Some(v) if isSensitive  => String.format("--%s ***********", longName)
+      case Some(Some(optionValue)) => prettyNameValue(optionValue)
+      case Some(None)              => String.format("--%s %s", longName, ReflectionUtil.SpecialEmptyOrNoneToken)
+      case Some(v)                 => String.format("--%s %s", longName, v.toString)
```

Comment:
Should we have case(s) here for collections?  I think we want `coll.mkString(" ")` as opposed to `coll.toString` which is what we'll get now.